app/src/main/java/one/mixin/android/vo/MessageItem.kt

```diff
@@ -155,7 +155,7 @@ data class MessageItem(
 
         try {
             if (type == MessageCategory.APP_CARD.name && appCardShareable == null) {
-                appCardShareable = appCardData?.shareable
+                appCardShareable = appCardData?.canShare
```

Comment:
this should be shareable?

shareable: ture -> display the share button
shareable: false -> hide the share button

shareable && actions.every(isValidShareUrl) == ture ->  share card body and actions

shareable && actions.every(isValidShareUrl) == false  => only share card body, remove the actions.